### DISCUSSION TITLE
Windows on Arm64 support

### DISCUSSION THREAD

**Discussion by Xazax-hun** [helix-editor/helix#8855] [Ideas]
It would be nice to release native binaries for Windows on Arm64. More and more such devices are started to popping up.

**Comment by pascalkuthe** [comment]
You can bud from source yourself. That is our approach for all nieche twrgets. The release binaries are really just a convenience for common targets. Helix is easy to build from source and we expect everyone with unusual needs to do so.

GH doesn't provided such CI runners so we cannot/will not provide such release binaries.
